### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by mgifford** [DSCoalition/digitalservicescoalition.org#34]
Worth running the site through WebAim's WAVE Toolbar to pick up contrast errors and missing alt text.   Will be a pretty simple fix but will make it more 508 compliant.
